lib/hashie/extensions/strict_key_access.rb

```diff
@@ -15,7 +15,7 @@ module Extensions
     #     >> hash[:cow]
     #       KeyError: key not found: :cow
     #
-    # NOTE: For googlers coming from Python to Ruby, this extension makes a Hash behave like a "Dictionary".
+    # NOTE: For googlers coming from Python to Ruby, this extension makes a Hash behave more like a "Dictionary".
```

Comment:
In reading the Hash documentation in Ruby standard library 2.2.3 I noticed that "Dictionary-like" is actually the very first sentence of the description of the Hash object.  So I changed this to say `more like a "Dictionary"`.

<img width="537" alt="hash documentation" src="https://cloud.githubusercontent.com/assets/19505/10712438/3d1d3f14-7a50-11e5-828c-701f08c8c5d5.png">